lib/pages/tabs/testchain/mainchain/withdrawal_bundle_tab_page.dart

```diff
@@ -170,3 +279,83 @@ class _WithdrawalViewState extends State<WithdrawalView> {
     return '$title to ${withdrawal.address}';
   }
 }
+
+const bundleViewWidth = 130.0;
+
+class ExpandedBundleView extends StatelessWidget {
+  final WithdrawalBundle bundle;
+
+  const ExpandedBundleView({
+    super.key,
+    required this.bundle,
+  });
+
+  // https://github.com/LayerTwo-Labs/testchain/blob/ba78df157fcb9f85d898f65db43c2842ab9473ff/src/policy/corepolicy.h#L31-L32
+  static const int maxStandardTxWeight = 400000;
+
+  // https://github.com/LayerTwo-Labs/testchain/blob/ba78df157fcb9f85d898f65db43c2842ab9473ff/src/policy/corepolicy.h#L25
+  static const int witnessScaleFactor = 4;
+
+  // https://github.com/LayerTwo-Labs/testchain/blob/ba78df157fcb9f85d898f65db43c2842ab9473ff/src/policy/corepolicy.h#L25
+  static const double maxWeight = (maxStandardTxWeight / witnessScaleFactor) / 2;
+
+  Map<String, dynamic> get _values => {
```

Comment:
Om du tar en kikk i dashboard_tab_page, kan dette gjøres generisk.


```
    return SailColumn(
      spacing: SailStyleValues.padding08,
      mainAxisAlignment: MainAxisAlignment.start,
      crossAxisAlignment: CrossAxisAlignment.start,
      children: decodedTX.keys.where((key) => key != 'walletconflicts').map((key) {
        return SingleValueContainer(
          label: key,
          value: decodedTX[key],
          width: width,
        );
      }).toList(),
    );
```

Reply:
If it is the suffixes you want, you could create a addSuffix() function that looks at the key, and appends a fitting suffix

Reply:
I'm just following the design spec 🫡